packages/create-redwood-app/src/create-redwood-app.js

```diff
@@ -798,10 +630,6 @@ async function createRedwoodApp() {
             `cd ${path.relative(process.cwd(), newAppDir)}`
           )}`
         )}`,
-        !yarnInstall &&
```

Comment:
I think the logic is flipped here. We should *always* include the instruction to run `yarn install` now, right?
(Hold changing anything, I'm still reviewing)

Reply:
Yep you're right I think oversight on my part in making this too fast